app/models/taxon_lineage.rb

```diff
@@ -179,13 +179,9 @@ def self.fetch_lineage_by_taxid(tax_map, pipeline_run_id)
     lineage_version = PipelineRun.find(pipeline_run_id).alignment_config.lineage_version
     lineage_by_taxid = {}
 
-    # Use pluck because it saved 5s on 10k rows in testing
-    # lineage_by_taxid[x.taxid] = x.as_json
-    TaxonLineage
-      .where(taxid: tax_ids).where("#{lineage_version} BETWEEN version_start AND version_end")
-      .pluck(:taxid, :species_taxid, :genus_taxid, :family_taxid)
-      .each do |arr|
-      lineage_by_taxid[arr[0]] = { taxid: arr[0], species_taxid: arr[1], genus_taxid: arr[2], family_taxid: arr[3] }
+    TaxonLineage.where(taxid: tax_ids).where("#{lineage_version} BETWEEN version_start AND version_end").each do |x|
```

Comment:
curious can this be rewritten to avoid it looking like a sql injection liability? Something like `.where("? BETWEEN version_start AND version_end", lineage_version)` or similar?

Reply:
👍 done

Reply:
yay thanks!